Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.yml

```diff
@@ -2002,9 +2028,34 @@ script:
       required: true
     description: Restores the network connection and removes IP addresses added to the exclusion list for the host with the specified agent ID.
     name: rsa-nw-endpoint-isolation-remove
-  dockerimage: demisto/python3:3.10.12.66339
+  - arguments: []
+    description: Returns the list of fields to map in outgoing mirroring. This command is only used for debugging purposes.
+    name: get-mapping-fields
+  - arguments:
+    - description: The remote incident ID.
+      name: id
+      required: true
+    - defaultValue: "0"
+      description: The UTC timestamp in seconds of the last update. The incident is only updated if it was modified after the last update time.
+      name: lastUpdate
+    description: Gets data from a remote incident. This method does not update the current incident, and should be used for debugging purposes only.
+    name: get-remote-data
+  - arguments:
+    - description: Date string representing the local time. The incident is only returned if it was modified after the last update time.
```

Comment:
What is the format of the date string?

Reply:
I don't know it's something native to XSOAR so I just implemented it the way it was explained in your documentation. So reading it, I think it can be multiple thing since It parsed using dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})